src/js/controller/captions.js

```diff
@@ -37,11 +37,13 @@ const Captions = function(_model) {
                     _addTrack(track);
                     loadFile(track, (vttCues) => {
                         _addVTTCuesToTrack(track, vttCues);
-                    }, (error) => {
-                        this.trigger(ERROR, {
-                            message: 'Captions failed to load',
-                            reason: error
-                        });
+                    }, (key, file, url, error) => {
+                        if (error.code === 404) {
```

Comment:
if error code is not 404 we still want to trigger an error. What happens if the code is 403?